packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -191,6 +195,16 @@ class DragUpdateDetails {
   /// Defaults to [globalPosition] if not specified in the constructor.
   final Offset localPosition;
 
+  /// A delta offset from the point where the drag initially contacted
```

Comment:
Would this be break since `offsetFromOrigin` has a default value, and `localOffsetFromOrigin` is nullable? One of the reasons i'm in favor of adding this, is that `LongPressMoveUpdateDetails` provides a similar API https://github.com/flutter/flutter/blob/9cb269c194b02d6b4688698460e4947f9f270ea0/packages/flutter/lib/src/gestures/long_press.dart#L166. Also that would avoid us having to save the `DragStartDetails` in our current text selection implementation, which is something I can't do with the text gestures project since the gesture recognizer mapping is defined in the classes initializer.